### ISSUE TITLE
sm_retakes_scramble_rounds

### ISSUE THREAD

**Issue by BrutalCSEjziponken** [splewis/csgo-retakes#185]
### Expected behavior
sm_retakes_scramble_rounds 3 is suppose to mix the teams after T wins 3 times. (?)

### Actual behavior
It does not seem to work at all. Nothing happens, just the normal team switches when CT wins.

### Steps to reproduce
- Plugin version: "CS:GO Retakes" (0.3.4+151) by splewis
- Sourcemod version: SourceMod Version: 1.10.0.6361
- Steps to reproduce (please be specific): Just play the game normally :P

**Comment by borzaka** [comment]
Yes, `sm_retakes_scramble_rounds 3` scrambles the players if the Ts won 3 times in a row. But if the CTs win before this 3 winning row, the counter resets, and a team switch occurs. With and exception an unlucky guy when the player numbers are odd.

**Comment by BrutalCSEjziponken** [comment]
Is it always the same guy? A player said he never switched from CT in 15 rounds in a row.

**Comment by borzaka** [comment]
No, not always the same guy. The one one doesn't got a kill in that round.
I don't thinks that possible what you are saying. He's exaggerating.
